dietpi/dietpi-survey

```diff
@@ -146,55 +144,42 @@ $(grep '=2' /DietPi/dietpi/.installed)
 _EOF_
 
 	#-----------------------------------------------------------------------------------
-	#Input mode: Force opted out, clear existing uploaded data
-	if (( $INPUT == 2 )); then
+	#Input mode: Send survey if opted in or empty file if opted out
+	if (( $INPUT == 1 )); then
 
-		OPTED_IN=0
 		Send_File
-		Write_Settings
-
-	#Input mode: Send survey if opted in
-	elif (( $INPUT == 1 )); then
-
-		if (( $OPTED_IN == 2 )); then
-
-			Send_File
-			Write_Settings
-
-		fi
 
 	else
 
 		G_WHIP_DEFAULT_ITEM=$OPTED_IN
 		G_WHIP_MENU_ARRAY=(
 
-			'2' 'Opt IN to DietPi-Survey.'
-			'0' 'Opt OUT and purge any existing survey data.'
+			'1' 'Opt IN to DietPi-Survey.'
+			'0' 'Opt OUT and purge my existing survey data.'
 
 		)
 
-		G_WHIP_MENU "DietPi-Survey would like to collect anonymous statistics about your device, version and installed software. \
+		G_WHIP_MENU "DietPi-Survey would like to collect anonymous statistics about your hardware, DietPi software and settings. \
 This allows us to focus development based on popularity. NO private data will be collected and NO ONE can identify you based on the data. \
 The data is sent via secured connection to our SFTP server and is stored there unreadable to the public upload user. \
-If you agree, your uploaded data will be automatically updated on every DietPi-Update and whenever you install software via DietPi-Software. \
-We regularly publish results of the collected data at: https://dietpi.com/survey
-Your personal upload file would currently look like this:
+If you agree, your uploaded data will be automatically updated on every DietPi-Update and DietPi-Software usage. \
+The current survey statistics can be reviewed at: https://dietpi.com/survey
+
+Your personal upload file would look like this:
 $(<$UPLOAD_FILENAME)
 
-Would you like to opt in for DietPi-Survey?"
+Would you join DietPi-Survey?"
```

Comment:
Minor language:
"Would you like to join the DietPi-Survey"

Reply:
@Fourdee 
Ah yes that's true, crippled sentence currently, whoopsie 🤣.